src/com/core/tictactoe/Game.java

```diff
@@ -0,0 +1,43 @@
+package com.core.tictactoe;
+
+public class Game {
+
+    Board board;
+    Player active;
+    Player passive;
+    CommandLineUI commandLineUI;
+
+    Game (CommandLineUI commandLineUI) {
+        this.board = new Board();
+        this.active = new Player("X");
+        this.passive = new Player("Y");
+        this.commandLineUI = commandLineUI;
+    }
+
+     void run() {
+        this.commandLineUI.greetUsers();
+        while (!this.board.isWon() && !this.board.isTie()) {
+            this.playTurn();
+        }
+        this.postGame();
+    }
+
+    private void playTurn() {
+        this.commandLineUI.showBoard(this.board);
+        int position = this.commandLineUI.getPosition(this.board, this.active);
+        this.board.putSignOnBoard(this.active, position);
+        this.switchPlayers();
```

Comment:
Love this!

Reply:
You use 'Collections'  sometimes I think?  There is potentially this: Collections.swap(array, index, index);

Reply:
Thank you!